lxd/patches.go

```diff
@@ -1465,6 +1713,13 @@ func patchStorageApiKeys(name string, d *Daemon) error {
 		if pool.Driver == "zfs" {
 			pool.Config["zfs.pool_name"] = pool.Config["source"]
 		} else if pool.Driver == "lvm" {
+			// On previous upgrade versions, "size" was set instead
+			// of "volume.size", so transfer the value and then
+			// unset it.
+			if pool.Config["size"] != "" {
+				pool.Config["volume.size"] = pool.Config["size"]
+				pool.Config["size"] = ""
+			}
```

Comment:
There will be another patch but I think it's also worth fixing this for users that run the upgrade for the first time.